cl/users/templates/includes/webhooks-modal.html

```diff
@@ -0,0 +1,20 @@
+{% load humanize %}
+<div id="webhook-modal"
+     class="modal hidden-print text-left"
+     role="dialog"
+     aria-hidden="true"
+     tabindex="-1">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span>
+        </button>
+        <h2 class="modal-title">Add a Webhook endpoint</h2>
```

Comment:
You capitalized webhook a lot in here. Let's keep them uncapitalized most of the time:

```suggestion
        <h2 class="modal-title">Add a webhook endpoint</h2>
```

Reply:
changed!